### PR TITLE
IE-3610 Modify template for sql database to allow for serverless tier

### PR DESCRIPTION
**Pull request by Maryonsin**
IE-3610 Modify template for sql database to allow for serverless configuration

### CODE REVIEW

**Review by havvo1 — APPROVED** [review]
Approved

File: resources/resourceAzureSqlDatabase/azuredeploy.jsonc [hunk 1]
```diff
@@ -14,43 +14,60 @@
...
             }
         },
-        "databaseCollation" : {
-            "type": "string",
-            "defaultValue" : "SQL_Latin1_General_CP1_CI_AS",
-            "metadata" : {
-                "description" : "The collation of the SQL Database."
-            }
-        },
+//       "databaseCollation" : {
```
Comment (lp-code) [inline resources/resourceAzureSqlDatabase/azuredeploy.jsonc:17]: Seems like the collation setting has been moved into the properties object. Delete it from here?

**Review by lp-code — COMMENTED** [review]
Should be fine once the commented part is removed!

**Comment by jlyshoel** [comment]
I've updated the test runner to also allow non-updated API versions in references, since we're already allowing outdated API usage. 

Before we merge this, we should test a bit more to make sure it works. Check PR in MOE repo.

**Review by jlyshoel — APPROVED** [review]

**Pull request merged**